light-base/src/sync_service.rs

```diff
@@ -41,7 +41,7 @@ use smoldot::{
     executor::host,
     libp2p::PeerId,
     network::{codec, service},
-    trie::{self, Nibble, prefix_proof, proof_decode},
+    trie::{self, Nibble, minimize_proof::minimize_proof, prefix_proof, proof_decode},
```

Comment:
```suggestion
    trie::{self, Nibble, minimize_proof, prefix_proof, proof_decode},
```

Also a nit-pick, but I prefer, and the whole code base only imports modules and not functions.